extensions/pkg/controller/worker/mapper.go

```diff
@@ -53,43 +49,24 @@ func MachineToWorkerMapper(predicates []predicate.Predicate) mapper.Mapper {
 }
 
 type machineSetToObjectMapper struct {
-	ctx            context.Context
-	client         client.Client
 	newObjListFunc func() client.ObjectList
 	predicates     []predicate.Predicate
 }
 
-func (m *machineSetToObjectMapper) InjectClient(c client.Client) error {
-	m.client = c
-	return nil
-}
-
-func (m *machineSetToObjectMapper) InjectStopChannel(stopCh <-chan struct{}) error {
-	m.ctx = contextutil.FromStopChannel(stopCh)
-	return nil
-}
-
-func (m *machineSetToObjectMapper) InjectFunc(f inject.Func) error {
```

Comment:
I guess we could now also get rid of this (probably in a separate PR though) https://github.com/gardener/gardener/blob/b1e7f278b6d3bc781631fbbb3b3292c816355fe2/pkg/controllerutils/predicate/or.go#L24-L25 
If I understand the reason for the removal of the `InjectFunc`

Reply:
Yes, good catch, thanks, I opened https://github.com/gardener/gardener/pull/6364